configuring.md

```diff
@@ -107,6 +107,9 @@ Some additional options can be configured globally by setting these system prope
 
 * **robolectric.enabledSdks** — Comma-separated list of SDK levels or names (e.g. `19, 21` or `KITKAT, LOLLIPOP`) which are enabled for this process. Only tests targetted at the listed SDKs will be run. By default, all SDKs are enabled.
 * **robolectric.offline** — Set to true to disable runtime fetching of jars.
+* **robolectric.usePreinstrumentedJars** — If true, Robolectric will use  instrumented jars to reduce instrumentation overhead. If changes are made to
```

Comment:
nit: remove empty space between "use" and "instrumented jars".

nit: I think description statement in one line is fine. IDE/Editor and Markdown render will help us to wrap long line.